<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EntryForm</class>
 <widget class="QWidget" name="EntryForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>80</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(26, 26, 26);</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QLineEdit" name="categoryLineEdit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>55</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>50000</width>
       <height>55</height>
      </size>
     </property>
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(51, 51, 51);
border-radius: 7px;
color: rgb(255, 255, 255);
font: 400 18pt &quot;Segoe UI&quot;;
qproperty-alignment: AlignCenter;</string>
     </property>
     <property name="placeholderText">
      <string>Ausgabe 1</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="spendingLineEdit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>50000</width>
       <height>55</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
background-color: rgb(74, 74, 74);
border-radius: 7px;
qproperty-alignment: AlignCenter;
color: rgb(229, 229, 229);
font: 600 18pt &quot;Segoe UI&quot;;
}
</string>
     </property>
     <property name="placeholderText">
      <string>Betrag</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="editButton">
     <property name="minimumSize">
      <size>
       <width>55</width>
       <height>55</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>55</width>
       <height>55</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
	background-color: rgb(74, 74, 74);
border-radius: 25px;
}
QPushButton:hover {
	
	background-color: rgb(63, 63, 63);
}
QPushButton:pressed {
	
	background-color: rgb(48, 48, 48);
}

QPushButton:disabled{
	
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../../../assets/recource.qrc">
       <normaloff>:/assets/pencil.png</normaloff>:/assets/pencil.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Maximum</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>50</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="deleteButton">
     <property name="minimumSize">
      <size>
       <width>55</width>
       <height>55</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>55</width>
       <height>55</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
	background-color: rgb(74, 74, 74);
border-radius: 9px;
font-size: 25px;
}
QPushButton:hover {
	
	background-color: rgb(63, 63, 63);
}
QPushButton:pressed {
	
	background-color: rgb(48, 48, 48);
}

QPushButton:disabled{
	
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../../../assets/recource.qrc">
       <normaloff>:/assets/recycle-bin.png</normaloff>:/assets/recycle-bin.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="submitButton">
     <property name="minimumSize">
      <size>
       <width>55</width>
       <height>55</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>55</width>
       <height>55</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">

QPushButton
{
	background-color: rgb(74, 74, 74);
	border-radius: 9px;
	font-size: 25px;
	color: rgb(85, 255, 0);
}
QPushButton:hover {
	
	background-color: rgb(63, 63, 63);
}
QPushButton:pressed {
	
	background-color: rgb(48, 48, 48);
}
QPushButton:disabled{
	
	//background-color: rgb(148, 148, 148);
}</string>
     </property>
     <property name="icon">
      <iconset resource="../../../assets/recource.qrc">
       <normaloff>:/assets/check.png</normaloff>:/assets/check.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
     <property name="shortcut">
      <string>Return</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../assets/recource.qrc"/>
 </resources>
 <connections/>
</ui>
